tests/test_unitNormMap.py

```diff
@@ -26,21 +27,30 @@ def test_UnitNormMapBasics(self):
             self.checkPersistence(unitnormmap)
 
             indata = np.array([
-                [1.0, 2.0, -6.0, 30.0, 1.0],
-                [3.0, 99.0, -5.0, 21.0, 0.0],
-                [-5.0, 3.0, -7.0, 37.0, 0.0],
-                [7.0, -23.0, -3.0, 45.0, 0.0],
+                [fullCenter[0], 1.0, 2.0, -6.0, 30.0, 1.0],
+                [fullCenter[1], 3.0, 99.0, -5.0, 21.0, 0.0],
+                [fullCenter[2], -5.0, 3.0, -7.0, 37.0, 0.0],
             ], dtype=float)[0:nin]
```

Comment:
This is kind of hard to follow. Maybe move `fullCenter` and `fullIndata` (?) outside the loop, to make it clearer that each iteration only needs a subset?